README.md

```diff
@@ -5,9 +5,7 @@ This is a macOS application, providing an interface to control an ATEM video swi
 
 ![atemOSC](https://github.com/danielbuechele/atemOSC/raw/master/atemOSC.jpg)
 
-The current version is built for Mac OS 10.12 SDK (as of version 2.4.7). A compiled and runnable version of the atemOSC is included which has been built against Blackmagic SDK 7.3 (as of version 2.5.2). 
```

Comment:
I think keep the `as of version x` on both, since they won't always be the same and it's a reminder to update them both.